PWGCF/FemtoDream/TableProducer/femtoDreamProducerTask.cxx

```diff
@@ -547,6 +602,70 @@ struct femtoDreamProducerTask {
         }
       }
     }
+
+    if (ConfIsActivateReso.value) {
+      for (auto iDaug1 = 0; iDaug1 < Daughter1.size(); ++iDaug1) {
+        for (auto iDaug2 = 0; iDaug2 < Daughter2.size(); ++iDaug2) {
+          // MC stuff is still missing, also V0 QA
+          // ALSO: fix indices and other table entries which are now set to 0 as deflaut as not needed for p-p-phi cf ana
+
+          ROOT::Math::PtEtaPhiMVector tempD1(Daughter1.at(iDaug1).pt(), Daughter1.at(iDaug1).eta(), Daughter1.at(iDaug1).phi(), ConfDaug1Daugh2ResoMass.value[0]);
+          ROOT::Math::PtEtaPhiMVector tempD2(Daughter2.at(iDaug2).pt(), Daughter2.at(iDaug2).eta(), Daughter2.at(iDaug2).phi(), ConfDaug1Daugh2ResoMass.value[1]);
+
+          ROOT::Math::PtEtaPhiMVector tempPhi = tempD1 + tempD2;
+
+          ResoRegistry.fill(HIST("AnalysisQA/Reso/InvMass"), tempPhi.M());
+
+          if ((tempPhi.M() >= ConfResoInvMassLowLimit.value) && (tempPhi.M() <= ConfResoInvMassUpLimit.value)) {
+
+            ResoRegistry.fill(HIST("AnalysisQA/Reso/InvMass_selected"), tempPhi.M());
+            ResoRegistry.fill(HIST("AnalysisQA/Reso/PtD1_selected"), Daughter1.at(iDaug1).pt());
+            ResoRegistry.fill(HIST("AnalysisQA/Reso/PtD2_selected"), Daughter2.at(iDaug2).pt());
+
+            childIDs[0] = 0;
+            childIDs[1] = 0;
+            std::vector<int> indexChildID = {0, 0};
```

Comment:
Be careful in here because `0` is a valid `id`. Check that this will not have further side effects